.github/workflows/correctness-linux.yaml

```diff
@@ -31,7 +31,7 @@ jobs:
         spack install googletest
         spack install cppcheck
         spack install yaml-cpp
-        spack install trilinos@master~epetra
+        spack install trilinos@master~epetra ^kokkos-kernels+blas+lapack
```

Comment:
Could you add this to the build recipes?

Reply:
Will do